packages/util/test/address.spec.ts

```diff
@@ -75,6 +75,14 @@ tape('Address', (t) => {
     st.end()
   })
 
+  t.test('should provide correct precompile check', (st) => {
+    const precompile = Address.fromString('0x0000000000000000000000000000000000000009')
+    st.true(precompile.isPrecompileOrSystemAddress(), 'should detect precompile address')
```

Comment:
Hmm this EIP is actually a little bit confusing. It does specify which address range could be assumed to be system precompiles. However, for access lists, we are only allowed to push precompiles which actually have code to the access lists. There was a bug in Besu where they also included BLS addresses to the access lists, which does not match the EIP. What is weird is that EIP2930 in combination with EIP1352 is not very consistent.

Reply:
Hmm, I'm just using this to *remove* precompile addresses, so there should be no problem? 🤔

Reply:
"which actually have code to the access lists"

What do you mean by that actually? 😀

Reply:
It means that once we setup the initial access list, we add the caller and the target to the access list, but we also add every precompile which has attached code to it. So, Besu made a mistake and also added the BLS precompile addresses. However, in Berlin, we do not have BLS, so there is no precompiled code at these BLS addresses and they should thus not be added to the access lists.